src/tvheadend/Subscription.cpp

```diff
@@ -173,12 +173,12 @@ void Subscription::SendUnsubscribe(void)
   Logger::Log(LogLevel::LEVEL_DEBUG, "demux successfully unsubscribed from channel id %d, subscription id %d", GetChannelId(), GetId());
 }
 
-bool Subscription::SendSeek(int time)
+bool Subscription::SendSeek(double time)
 {
   /* Build message */
   htsmsg_t *m = htsmsg_create_map();
   htsmsg_add_u32(m, "subscriptionId", GetId());
-  htsmsg_add_s64(m, "time",           static_cast<int64_t>(time * 1000LL));
+  htsmsg_add_s64(m, "time",           time * 1000LL);
```

Comment:
Why no cast anymore? `htsmsg_add_s64` still expects an `int64_t`, not a double, right?